### DISCUSSION TITLE
Import types only for front-end code

### DISCUSSION THREAD

**Discussion by stijnvanderlaan** [prisma/prisma#2969] [Q&A] [answered]
## Problem

I get the next error when I try to import a enum at the front-end of my code:
```
Module not found: Error: Can't resolve 'child_process' in '/Users/stijn/Workspace/demonsters/videochatly/node_modules/@prisma/client/runtime'
```

## Suggested solution
It would be nice I could import the types only:
```js
import { Status } from '@prisma/client/types'
```

**Comment by ryands17** [comment, marked as answer]
Hey @stijnvanderlaan :wave: 
Unfortunately Prisma cannot be loaded in the frontend so you would either have to replicate those types or use a type generator like [this](https://graphql-code-generator.com/) if you're using GraphQL.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
